unit_tests/kernels/UnitTestKernelUtils.C

```diff
@@ -343,6 +402,12 @@ init_trigonometric_field(
     funcPtr = &TrigFieldFunction::dkdx;
   else if (fieldName == "specific_dissipation_rate")
     funcPtr = &TrigFieldFunction::sdr;
+  else if (fieldName == "gamma_transition") // added: transition
```

Comment:
remove all `// added: transition" comments. Here and in other files.

Reply:
Thanks. I forgot to remove these comments. I will do that.

Reply:
done